bok_choy/page_object.py

```diff
@@ -534,6 +534,18 @@ def axs_audit_rules_to_run(self):
         """
         return []
 
+    def axs_audit_rules_to_ignore(self):
+        """
+        List of rules to ignore for accessibility errors on the page.
+        See https://github.com/GoogleChrome/accessibility-developer-tools/tree/master/src/audits
+
+        E.g. return ['badAriaAttributeValue']
+        An empty list means to run rules as defined by axs_audit_rules_to_run.
+        Otherwise, if rules are listed here, they will be ignored even if
+        they are specified in axs_audit_rules_to_run.
+        """
+        return None
```

Comment:
Sort of...  it's a bit more complicated I think.  I see it as `[]` having different meanings, actually.

I think its more like  `rules_actually_run = set(axs_audit_rules_to_run) - set(axs_audit_rules_to_ignore)`. In `axs_audit_rules_to_ignore`, both [] and None are considered empty sets.  But, in `axs_audit_rules_to_run`,  `[]` (which is normally considered empty) is considered "the set of all rules" and None is considered an empty set.

Would it be less confusing to change it to `[]` here and not document the option of setting it to `None` though?